include/fmt/base.h

```diff
@@ -1792,9 +1792,9 @@ template <typename T> class buffer {
 };
 
 struct buffer_traits {
-  explicit buffer_traits(size_t) {}
-  auto count() const -> size_t { return 0; }
-  auto limit(size_t size) -> size_t { return size; }
+  FMT_CONSTEXPR explicit buffer_traits(size_t) {}
+  FMT_CONSTEXPR auto count() const -> size_t { return 0; }
+  FMT_CONSTEXPR auto limit(size_t size) -> size_t { return size; }
```

Comment:
These can be `constexpr`. `FMT_CONSTEXPR` is only for cases when C++14 relaxed `constexpr` is needed.

Reply:
Okay my bad, hadn't realized the distinction. I think all the changes are now `constexpr` where they can be.